fixtures/pages-functions-app/wrangler.toml

```diff
@@ -0,0 +1,48 @@
+# Generated by Wrangler on Mon Mar 25 2024 09:33:15 GMT+0000 (Greenwich Mean Time)
```

Comment:
So you used the download command to generate this?
But is this comment relevant now here?

Reply:
I've removed this file